public/tsconfig.json

```diff
@@ -1,25 +1,37 @@
 {
   "compileOnSave": false,
   "compilerOptions": {
-    // "rootDir": ".",
     "sourceMap": true,
     "declaration": false,
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "importHelpers": true,
-    "target": "es2015",
+    "target": "ES2022",
```

Comment:
Same question as admin, but more critical given more diverse user base - will this affect browser compatibility?

Reply:
While migrating to Angular 15, it automatically correct this and also drops previous ".browserslistrc". Also I think I read somewhere that Angular no longer supports IE. If you add this back to 2015 (ES6), Angular will give a warning. And even with ES2022, I think Angular by default also support previous versions as in previous ".browserslistrc". So they seem to have good reason for doing that. We still can add this ".browserslistrc" but probably is not necessary anymore.
![image](https://github.com/bcgov/nr-fom/assets/81595625/26d02e42-4a6e-4e5d-bae8-6d2ade9c0834)

Reply:
This was discussed, agree we keep what default configuration set from Angular.